plugins/inventory/terraform_state.py

```diff
@@ -10,10 +10,11 @@
   - Aubin Bikouo (@abikouo)
 short_description: Builds an inventory from resources created by cloud providers.
 description:
-  - This plugin works with an existing state file to create an inventory from resources created by cloud providers.
-  - The plugin accepts a Terraform backend config to an existing state file or a path to an existing state file.
-  - Uses a YAML configuration file that ends with terraform_state.(yml|yaml).
-  - To read the state file command ``Terraform show`` is used.
+  - This plugin works by making a temporary directory, writing a C(main.tf) file with the configured backend, then running C(terraform init),
+    followed by C(terraform show) in JSON format to read state.
```

Comment:
These are implementation details and not something I think should appear in the module docs. There's currently a PR (#153) that changes this.

Reply:
Ok, I obviously disagree.  The current way it works should be stated somewhere (perhaps in NOTES?)  I think considering the potential for danger in running something like `init`, it should be quite clear what it currently does.  It currently states "`terraform state` is used", so I felt it needed fleshing out.

> There's currently a PR (#153) that changes this.

If the PR changes this, then the doco can change at that point?

The parameters section documents things that implement changes, so :shrug: